app/Services/SiteService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Services;
+
+use App\Repositories\CategoryRepository;
+use Illuminate\Http\Request;
+use App\Repositories\ProductRepository;
+use Session;
+
+class SiteService
+{
+    protected $productRepo;
+    protected $cateRepo;
+
+    public function __construct(ProductRepository $productRepo,
+                                CategoryRepository $cateRepo)
+    {
+        $this->productRepo = $productRepo;
+        $this->cateRepo = $cateRepo;
+    }
+
+    public function rating(Request $request)
+    {
+        $params = [
+            'count_rating' => $request->count_rating,
+            'star_rating' => $request->star_rating,
+            'star' => $request->star,
+            'score_rating' => $request->score_rating
+        ];
```

Comment:
dùng phương pháp này để lấy params 
https://laravel.com/docs/8.x/requests#determining-if-input-is-present